dir.props

```diff
@@ -69,13 +69,13 @@
     <DebugSymbols Condition="'$(DebugSymbols)' == ''">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == ''">false</Optimize>
     <DebugType Condition="'$(DebugType)' == ''">full</DebugType>
-    <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
+    <DefineConstants>$(DefineConstants),DEBUG,TRACE</DefineConstants>
```

Comment:
Have you verified that both the native and Roslyn C# compiler supports comma? We still support both.